pyrometheus/chem_expr.py

```diff
@@ -173,26 +173,26 @@ def _zeros_like(argument):
 def transport_polynomial_expr(c, n, t):
     """Generate code for transport polynomials
     :returns: Transport polynomial expression with coefficients c in terms of
-    the temperature t as a :class:`pymbolic.primitives.Expression`. For `thermal_conductivity`, `n = 1`, while for `viscosity` `n = 2`
+    the temperature t as a :class:`pymbolic.primitives.Expression`. For
+    `thermal_conductivity`, `n = 1`, while for `viscosity` `n = 2`
     """
-    log = p.Variable("log")
     assert len(c) == 5
     return (
         (
-            c[0] +
-            c[1] * p.Variable("log")(t) +
-            c[2] * p.Variable("log")(t) ** 2 +
-            c[3] * p.Variable("log")(t) ** 3 +
-            c[4] * p.Variable("log")(t) ** 4
+            c[0]
+            + c[1] * p.Variable("log")(t)
+            + c[2] * p.Variable("log")(t) ** 2
+            + c[3] * p.Variable("log")(t) ** 3
+            + c[4] * p.Variable("log")(t) ** 4
         )**n
     )
 
 
 def wilke_mixture_rule_expr(sol: ct.Solution, sp, x, mu):
```

Comment:
Rename to `viscosity_mixture_rule_expr`, or `viscosity_mixture_rule_wilke_expr`.

Reply:
Done. Renamed to "viscosity_mixture_rule_wilke_expr".